fe/fe-core/src/main/java/com/starrocks/common/util/KafkaUtil.java

```diff
@@ -188,7 +190,7 @@ private PProxyResult sendProxyRequest(PProxyRequest request) throws UserExceptio
                 // get info
                 request.timeout = Config.routine_load_kafka_timeout_second;
                 Future<PProxyResult> future = BackendServiceClient.getInstance().getInfo(address, request);
-                PProxyResult result = future.get(Config.routine_load_kafka_timeout_second, TimeUnit.SECONDS);
+                PProxyResult result = future.get(GET_INFO_TIMEOUT, TimeUnit.SECONDS);
```

Comment:
what will happen when this rpc time cost more than routine_load_kafka_timeout_second